{{cookiecutter.project_name}}/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Shutting up pylint comlaining about D104."""
```

Comment:
Let's make this slightly more informative 🙃 

```suggestion
"""Unit tests for the project."""
```